snippets.ts

```diff
@@ -0,0 +1,79 @@
+// let msg = `Collected ${reaction.emoji.name} from user ${user.username}`
```

Comment:
Can we remove this file if it's not in use?

Reply:
removed